tests/test_api.py

```diff
@@ -780,9 +780,9 @@ def test_generate_manifest_file_based_annotations(
 
         # make sure Filename, entityId, and component get filled with correct value
         assert google_sheet_df["Filename"].to_list() == [
-            "schematic - main/TestDataset-Annotations-v3/Sample_A.txt",
-            "schematic - main/TestDataset-Annotations-v3/Sample_B.txt",
-            "schematic - main/TestDataset-Annotations-v3/Sample_C.txt",
+            "schematic - main/TestDatasets/TestDataset-Annotations-v3/Sample_A.txt",
```

Comment:
I think this integration test is enough, it probably should've been these values to begin with

Reply:
I agree - This along with other tests (I verified we have coverage on the new changes) should be sufficient.

Reply:
I guess one thought is should we add a test if the datasetId is the projectid so that it triggers: `if dirpath[0].startswith(f"{project_name}/")`

Reply:
It does exist - I verified that when running `tests/test_api.py::TestSynapseStorage::test_get_dataset_files[None-True]` it hits this path